### DISCUSSION TITLE
auto save to database

### DISCUSSION THREAD

**Discussion by b4guette1** [jacobwojoski/dice-stats#42] [Ideas]
could there be a toggle for rolls to be autosaved to the database?

**Comment by jacobwojoski** [comment]
Make an Issue and I could get to it eventually. Making the game save data every roll isnt the best performance wise. Saving and pulling data from the DB is kinda slow and the documentation says it could impact players performance more then other things. But making it a setting isn't a bad idea so people could choose to activate it if its not a big performance hit for them.

**Comment by jacobwojoski** [comment]
Added
